staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -1700,6 +1700,10 @@ func (az *Cloud) reconcileLoadBalancerRule(
 			loadDistribution = network.LoadDistributionSourceIP
 		}
 
+		tcpReset := enableTCPReset
+		if port.Protocol != v1.ProtocolTCP {
+			tcpReset = nilTCPReset
```

Comment:
This can just be `tcpReset = nil` and you can get rid of `nilTCPReset`?

Reply:
Good catch. Updated.